### DISCUSSION TITLE
Node-Renaming all its entities’ IDs in Home Assistant

### DISCUSSION THREAD

**Discussion by Ace0510** [zachowj/hass-node-red#277] [Q&A]
I use Node-Red add-on with 21 entities. Yesterday the ID of all of them changed by adding to all the entities ID a suffix “_2”. This is the second time this has happened. The problem is that the original entity IDs are still registered. Any ideas why this happened? How can delete the old registered IDs?

**Comment by zachowj** [comment]
Are you talking about entities created from NR? What version of the HA nodes are you running? <https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/diagnostics.html>

Are the entities inside subflows?
